waiverdb/templates/permissions.html

```diff
@@ -14,28 +14,25 @@
                         <th>Name</th>
                         <th>Maintainers</th>
                         <th>Testcases</th>
-                        <th>Users/Groups</th>
+                        <th>Groups</th>
```

Comment:
When I was doing this page I fought two opposite problems: I wanted to avoid names wrapping, so user can be sure that one line always contains exactly one user name, group name or testcase name. On other hand I wanted to avoid horizontal scrolling, which is considered as a very user-unfriendly thing. That's why I removed description (see below) and that's why I put users and groups together (most of the rules contain either users or groups but if you have a combination of rules containing groups and users in your table, you will need horizontal space for both columns).